dvc/schema.py

```diff
@@ -16,10 +16,11 @@
     StageParams.PARAM_ALWAYS_CHANGED: bool,
 }
 
+DATA_SCHEMA = {Required("md5"): output.CHECKSUM_SCHEMA, Required("path"): str}
```

Comment:
`md5` is not the only checksum we have. It might also be `etag` for stuff like external s3 deps/outputs and so on.

Reply:
Missed it. Thanks.

Reply:
It's fixed by a6a7bc1.